### PR TITLE
Fixed number precision to 2 digits after decimal point in ImpactTable

### LINKED ISSUE #85: ImpactTable: Limit the number of decimal points
The impact table used in the review and report pages currently doesn't limit the number of decimal points displayed for values in the initial, latest and delta columns.

Adjust this so that numbers are shown with up to 2 decimal points.

### PR DESCRIPTION
**Pull request by nikw92**
Closes #85

### CODE REVIEW

File: src/app/components/ImpactTable/index.tsx [hunk 1]
```diff
@@ -29,9 +29,9 @@ class ImpactTable extends React.Component<IProp, any> {
     return (
       <Table.Row key={r.name}>
         <Table.Cell>{r.name}</Table.Cell>
-        <Table.Cell>{r.first}</Table.Cell>
-        <Table.Cell>{r.last}</Table.Cell>
-        <Table.Cell>{d === undefined ? 'N/A' : d}</Table.Cell>
+        <Table.Cell>{r.first.toPrecision(3)}</Table.Cell>
+        <Table.Cell>{r.last.toPrecision(3)}</Table.Cell>
```
Comment (drimpact) [inline src/app/components/ImpactTable/index.tsx]: first and last here can be `number|undefined`. What happens if you run `.toPrecision(3)` on `undefined`?

**Comment by nikw92** [comment]
That would result in an error. I'll make it 'N/A' in that case just like delta.
3. Oct 2017 12:11 by notifications@github.com:

**Comment by drimpact** [comment]
Also should this be `toFixed` rather than `toPrecision`

**Comment by nikw92** [comment]
Made the requested changes.

**Comment by nikw92** [comment]
Accidentally added a commit not related to this issue. Reverted it so that it can be dealt with in a separate pull request.

**Pull request merged**